cluster/store/store_log.go

```diff
@@ -1,45 +1,83 @@
 package store
 
-import "github.com/hashicorp/raft"
+import (
+	"fmt"
+	"github.com/ByteStorage/FlyDB/config"
+	"github.com/ByteStorage/FlyDB/lib/datastore"
+	"github.com/hashicorp/raft"
+)
 
-// raftLog implements raft.LogStore interface, we can use it to store raft logs
-// how to store raft logs? we can use FlyDB/RocksDB/LevelDB/BoltDB to store raft logs
-// maybe we can use FlyDB
-type raftLog struct {
-	// implement me
-}
+// DataStoreFactory is a function type that creates a new instance of a raft.LogStore.
+// It takes a configuration map as input and returns the created LogStore or an error if the creation fails.
+type DataStoreFactory func(conf config.Options) (raft.LogStore, error)
```

Comment:
May be the `config.Options` should be replaced with Config in `config/cluster_options.go`?